packages/dds/tree/api-report/tree.alpha.api.md

```diff
@@ -396,6 +396,20 @@ export type ImplicitFieldSchema = FieldSchema | ImplicitAllowedTypes;
 // @alpha
 export function independentInitializedView<const TSchema extends ImplicitFieldSchema>(config: TreeViewConfiguration<TSchema>, options: ForestOptions & ICodecOptions, content: ViewContent): TreeViewAlpha<TSchema>;
 
+// @alpha
+export function independentTreeAlpha<const TSchema extends ImplicitFieldSchema>(options?: ForestOptions & (({
```

Comment:
Nit: our guidelines call for functions to generally be named using a verb phrase. Should this be `createIndependentTreeAlpha` (or something) instead?

Reply:
done